src/Data/Nat/Divisibility/Core.agda

```diff
@@ -40,10 +40,23 @@ open _∣_ using (quotient) public
 _∤_ : Rel ℕ 0ℓ
 m ∤ n = ¬ (m ∣ n)
 
--- smart constructor
+-- Smart constructor
 
 pattern divides-refl q = divides q refl
 
+------------------------------------------------------------------------
+-- Restricted divisor relation
+
+-- Relation for having a non-trivial divisor below a given bound.
+-- Useful when reasoning about primality.
+record _HasNonTrivialDivisorLessThan_ (m n : ℕ) : Set where
```

Comment:
Should `m` and `n` be exchanged in order to be congruent with the meaning intended by the name of this type?

Should we (now) make this the `Bundle`, for which a `Structure` `IsBoundedStrictDivisor` is definable with the 'obvious' meaning?

Reply:
One outstanding issue: fixity declaration for this parametrised `record` definition?